openff/qcsubmit/common_structures.py

```diff
@@ -400,6 +415,44 @@ def to_string(self) -> str:
         Cavity {{{cavity_str}}}"""
         return pcm_string
 
+    def add_keywords(self, keyword_data: dict) -> dict:
+        keyword_data["pcm"] = True
+        keyword_data["pcm__input"] = self.to_string()
+        return keyword_data
+
+
+class DDXSettings(_BaseSolvent):
+    """
+    A simple settings class for the ddx solvent model to be used with Psi4
+    """
+
+    ddx_model: Literal["pcm", "cosmo"] = Field(
+        "pcm", description="The solvation model to use."
+    )
+    ddx_radii_scaling: PositiveFloat = Field(
+        1.1,
+        description="The scaling factor for the cavity spheres this also depends on the radii set chosen.",
```

Comment:
not blocking, very minor
```suggestion
        description="The scaling factor for the cavity spheres. This also depends on the radii set chosen.",
```